integration-cli/docker_cli_volume_test.go

```diff
@@ -212,7 +212,7 @@ func (s *DockerSuite) TestVolumeCliRm(c *check.C) {
 func (s *DockerSuite) TestVolumeCliNoArgs(c *check.C) {
 	out, _ := dockerCmd(c, "volume")
 	// no args should produce the cmd usage output
-	usage := "Usage:	docker volume [OPTIONS] [COMMAND]"
+	usage := "Usage:	docker volume COMMAND"
```

Comment:
This change was because `docker volume` doesn't actually have any (meaningful) options. It has `--help`, but that is inherited by all commands. So the new code doesn't print the `[OPTIONS]` section.

Any command with options will include the `[OPTIONS]` text as part of its usage.

I could add it explicitly here, but it seems more correct to omit it.